README.md

```diff
@@ -0,0 +1,77 @@
+asb-installer
+======================
+
+An Ansible Playbook Bundle (APB) for deploying a the [Ansible Service Broker (ASB)](https://github.com/openshift/ansible-service-broker).
+
+## Requirements
+An OpenShift cluster. (e.g. `oc cluster up`)
+
+User must be logged in with cluster-admin privileges. (e.g. `oc login -u system:admin`)
```

Comment:
Line 9 essentially says the same thing as line 17. You probably don't need this line.